README.md

```diff
@@ -116,6 +117,8 @@ You can debug a PMTiles file using <https://protomaps.github.io/PMTiles>.
 
 There's a manual step required to generate `--wards`, `--census_output_areas`, and `--imd`. See the comment in the code.
 
+For `--cycle_paths`, you'll need about 20GB of RAM, until we switch to a streaming JSON parser.
```

Comment:
https://pypi.org/project/ijson/ could help, but we need to set up poetry or venv for dependencies